tools/benchmark/cmd/watch.go

```diff
@@ -194,7 +194,7 @@ func doWatch(stream v3.Watcher, requests <-chan string) {
 
 func recvWatchChan(wch v3.WatchChan) {
 	for range wch {
-		if atomic.LoadInt32(&nrRecvCompleted) == int32(eventsTotal) {
+		if atomic.LoadInt32(&nrRecvCompleted) >= int32(eventsTotal) {
```

Comment:
The key space size defaults to 1 so each watcher will watch on the same key; one put will result in several watch events.